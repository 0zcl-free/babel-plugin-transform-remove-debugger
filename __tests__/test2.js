const babel = require("@babel/core")
const plugin = require('../src/index')

it('test2', () => {
  const example = `
  console.log('a');debugger;
  console.log('b')
  debugger;
  console.log('c')
  `
  const { code } = babel.transform(example, {plugins: [plugin]});
  expect(code).toMatchSnapshot();
})