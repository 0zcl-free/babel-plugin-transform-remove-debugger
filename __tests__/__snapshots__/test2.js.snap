// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test2 1`] = `
"console.log('a');
console.log('b');
console.log('c');"
`;
